recipes/precession/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "precession" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 792bf8e3441c4ea012b91d278eca00bb134bcb94a601716491884935648d7bec
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numpy
+    - parmap >=1.5.1
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - precession
+    - precession.test
```

Comment:
```suggestion
test:
  imports:
    - precession
    - precession.test
  commands:
    - pip check
  requires:
    - pip
```

Reply:
The `pip check` stage that Grayskull adds is usually a useful cross-check that the dependency tree checks out.